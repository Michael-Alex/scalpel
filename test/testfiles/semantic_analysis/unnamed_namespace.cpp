namespace
{
	class c{};
	int i;
}

c c0;
int i;

