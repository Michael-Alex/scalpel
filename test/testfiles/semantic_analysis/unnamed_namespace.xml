<namespace id="n0">
 <unnamed_namespace>
  <class name="c" id="c0">
   <destructor access="public" defined="false">
   </destructor>
  </class>
  <variable name="i">
   <type>
    <fundamental_type type="int"/>
   </type>
  </variable>
 </unnamed_namespace>
 <variable name="c0">
  <type>
   <class id="c0"/>
  </type>
 </variable>
 <variable name="i">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
</namespace>
