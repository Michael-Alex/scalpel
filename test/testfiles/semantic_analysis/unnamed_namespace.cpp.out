<namespace id="0">
 <unnamed_namespace>
  <class name="c" id="0">
   <destructor access="public" defined="false">
   </destructor>
  </class>
  <variable name="i">
   <type>
    <fundamental_type type="int"/>
   </type>
  </variable>
 </unnamed_namespace>
 <variable name="c0">
  <type>
   <class id="0"/>
  </type>
 </variable>
 <variable name="i">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
</namespace>
